platform/ext/target/nordic_nrf/nrf5340/target_cfg.h

```diff
@@ -45,6 +45,23 @@ struct memory_region_limits {
  */
 int32_t spu_init_cfg(void);
 
+/**
+ * \brief Configures the Security Protection Unit (peripheral security).
```

Comment:
"Security" should be "System". Otherwise this is just a suggestion I find more readable.
```suggestion
 * \brief Configures peripheral permissions via the System Protection Unit.
```

Reply:
Nice suggestion, really. Updated.

Reply:
Updated also the spu_init_cfg documentation to match this new suggestion